wiki/Tournaments/CFC/2018/en.md

```diff
@@ -0,0 +1,223 @@
+
+## Tournament Schedule
+
+| Event              | Timestamp                                  |
+|-------------------:|--------------------------------------------|
+| Registration Phase | 2018-03-11/2018-04-01 |
+| Group Stage        | 2018-04-07/2018-04-09 |
+| Quarterfinals + Loser's Bracket Round 1 | 2018-04-14/2018-04-15 |
+| Semifinals + Loser's Bracket Round 2 | 2018-04-21/2018-04-22  |
+| Finals - Week 1    | 2018-04-28/2018-04-29  |
+| Finals - Week 2    | 2018-05-05/2018-05-06 |
+
+## Prizes
+
+| Place                                             | Prize(s)                             |
+|-:-:-------------------------------------------------|---------------------------------------|
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge + 8 months of supporter |
+
+## Organization
+
+The Catch French Cup 2018 is run by various french community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| ------------ | -------------- |
+| Host | ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) |
+| GFX  | ![][flag_FR] [Lotchidych](https://osu.ppy.sh/users/8375240)   |
+| Map Selectors | ![][flag_FR] [Rougoku](https://osu.ppy.sh/users/6900166), ![][flag_FR] [Holloh](https://osu.ppy.sh/users/7612994)  |
+| Referees | ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_FR] [Ryumi](https://osu.ppy.sh/users/6596270), ![][flag_FR] [Rsuiono](https://osu.ppy.sh/users/9266902), ![][flag_FR] [TLQ_Yoshii](https://osu.ppy.sh/users/7157133), ![][flag_FR] [300BPM JUMPER](https://osu.ppy.sh/users/1893667), ![][flag_FR] [Purettsu Eru](https://osu.ppy.sh/users/1542565) |
+| Streamer | ![][flag_FR] [Purettsu Eru](https://osu.ppy.sh/users/1542565), ![][flag_FR] [FulgurentKille](https://osu.ppy.sh/users/4096322), ![][flag_FR] [300BPM JUMPER](https://osu.ppy.sh/users/1893667), ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_FR] [Star Sapphire](https://osu.ppy.sh/users/2953898) | 
+| Commentators | ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_FR] [FulgurentKille](https://osu.ppy.sh/users/4096322), ![][flag_FR] [Corn-](https://osu.ppy.sh/users/4089232), ![][flag_FR] [Orazeit](https://osu.ppy.sh/users/8335913), ![][flag_FR] [Star Sapphire](https://osu.ppy.sh/users/2953898), ![][flag_FR] [4L-Chan 3 Porte](https://osu.ppy.sh/users/7253987) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/714536)
+- [Livestream](https://www.twitch.tv/osufrlive)
+- **[Main Spreadsheet](https://docs.google.com/spreadsheets/d/1AwvOj_yBpcrB5sLEtRpEE7Wq7ZTVSZRncKzPjs7T-xQ/edit?usp=sharing)**
+
+-----------------------------------
+
+<!-- ![Podium](podium.jpg) -->
+
+<!-- ![Bracket](bracket.jpg) -->
+
+---------------------------------------
+
+## Participants
+
+| Group | Top | High | Low | Unseeded |
+| ------- | ------- | ------- | --------- |
```

Comment:
This table is still broken. Copy and replace.
| ------- | ---- | ----- | ----- | ----------- |